### PR TITLE
reading the proposed Total Board Power for RX 7000 Cards

### PR DESCRIPTION
**Pull request by NecromancerForLife**
Values are too high maybe a coorection is needed

### CODE REVIEW

File: Aga.Controls/Aga.Controls.csproj [hunk 1]
```diff
@@ -9,6 +9,7 @@
     <FileVersion>1.7.0.0</FileVersion>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <OutputPath>..\bin\$(Configuration)\</OutputPath>
+    <GenerateResourceUsePreserializedResources>true</GenerateResourceUsePreserializedResources>
```
Comment (PhyxionNL) [inline Aga.Controls/Aga.Controls.csproj:12]: Can you undo this?

File: Aga.Controls/Aga.Controls.csproj [hunk 2]
```diff
@@ -118,4 +119,7 @@
     <Compile Remove="Tree\ColumnCollection.cs" />
     <Compile Remove="Tree\NativeMethods.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="System.Resources.Extensions" Version="7.0.0" />
+  </ItemGroup>
```
Comment (PhyxionNL) [inline Aga.Controls/Aga.Controls.csproj:124]: Can you undo this?

File: Aga.Controls/PerformanceAnalyzer.cs [hunk 3]
```diff
@@ -19,6 +19,7 @@ public string Name
 			}
 
 			private int _count = 0;
+
```
Comment (PhyxionNL) [inline Aga.Controls/PerformanceAnalyzer.cs:22]: Can you undo this?

File: LibreHardwareMonitor/LibreHardwareMonitor.csproj [hunk 4]
```diff
@@ -13,6 +13,7 @@
     <OutputPath>..\bin\$(Configuration)\</OutputPath>
 	  <LangVersion>latest</LangVersion>
     <PlatformTarget>AnyCPU</PlatformTarget>
+    <GenerateResourceUsePreserializedResources>true</GenerateResourceUsePreserializedResources>
```
Comment (PhyxionNL) [inline LibreHardwareMonitor/LibreHardwareMonitor.csproj:16]: Can you undo this?

**Review by PhyxionNL — CHANGES_REQUESTED** [review]

File: LibreHardwareMonitor/LibreHardwareMonitor.csproj (hunk 4 above)
Comment (NecromancerForLife) [reply LibreHardwareMonitor/LibreHardwareMonitor.csproj]: done

File: Aga.Controls/PerformanceAnalyzer.cs (hunk 3 above)
Comment (NecromancerForLife) [reply Aga.Controls/PerformanceAnalyzer.cs]: done

File: Aga.Controls/Aga.Controls.csproj (hunk 2 above)
Comment (NecromancerForLife) [reply Aga.Controls/Aga.Controls.csproj]: done

File: Aga.Controls/Aga.Controls.csproj (hunk 1 above)
Comment (NecromancerForLife) [reply Aga.Controls/Aga.Controls.csproj]: done

**Pull request merged**